### PR TITLE
chore: remove avl

### PR DESCRIPTION
**Pull request by u2**
https://github.com/cryptape/cita-common/issues/199

### CODE REVIEW

**Review by keroro520 — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Generated by Cargo
 # will have compiled files and executables
 /target/
+*/target/
```
Comment (chaoticlonghair) [inline .gitignore]: First, if a rust project was in a workspace, the build files are always in the top `target` directory, so, I think this line is unnecessary.

And, if you really want to add the non-top `target` directory, you can just change the `/target/` to `target/`.

Comment (keroro520) [reply .gitignore]: `*/target/` == `target/` ?

Comment (chaoticlonghair) [reply .gitignore]: `target/ == /target/ + /*/target/ + /*/*/target/ +  /*/*/*/target/ + ...`

Comment (u2) [reply .gitignore]: While running `cargo bench --features "sha3hash` under the `cita-common-benches`, cargo would create a `target` dir under the `cita-common-benches`. I did not dive into the reason.

Comment (chaoticlonghair) [reply .gitignore]: Oh, I see, [`criterion`] stores the benchmark results into that `target` directory.

[`criterion`]: https://crates.io/crates/criterion

Comment (chaoticlonghair) [reply .gitignore]: 1. `target/` include `/target/`.
2. I think add `criterion/` is enough and more suitable.

Comment (u2) [reply .gitignore]: I think just `target/` is better.

Comment (chaoticlonghair) [reply .gitignore]: OK!

**Review by chaoticlonghair — APPROVED** [review]

**Pull request merged**